assets/script.js

```diff
@@ -123,10 +123,55 @@ function displayRecipes(url) {
       let priceEl = $("<li>").text(`Price per serving: ${pricePerServing}`);
       let timeEl = $("<li>").text(`Minutes to prepare meal: ${readyInMinutes}`);
       let caloriesEl = $("<li>").text(`Calories: ${calories}`);
-
-      recipeDiv.append(headerEl, recipeEl);
+      
+      // Modal button and properties added here
+        let buttonEl = $("<button>")
+        .addClass("btn btn-secondary modal-btns")
+        // add attribute as an identifier for modal button to use to get recipie info from API
+        .attr("id", mealID)
+        .text("Cooking instructions");
+
+      recipeDiv.append(headerEl, recipeEl, buttonEl);
       recipeEl.append(priceEl, timeEl, caloriesEl);
       recipesContainerDiv.append(recipeDiv);
     }
   });
 }
+
+
+
+
+// Code to control opening and closing of modal
+
+// Get the modal div from HTML
+var modal = $("#myModal");
+
+// Get the button that opens the modal
+var modalBtn = $(".modal-btn");
```

Comment:
I don't think this is used anywhere? The cooking instructions buttons won't exist at this stage

Reply:
Yes you are write, I put it in initially before switching to the $(document).on("click",".modal-btns",openModal) when I realised it wouldn't work, but forgot to take those lines out. I'll remove lines 149 and 150.